services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java

```diff
@@ -2019,6 +2040,28 @@ public Builder snappingIncludeClosuresList(@Nullable List<Boolean> snappingClosu
     @NonNull
     public abstract Builder experimental(@Nullable Experimental experimental);
 
+    /**
+     * Use this method to add request parameters,
+     * which are not present in the model yet but are supported on the Directions API,
+     * to an url generated by `RouteOptions#toUrl`.
```

Comment:
```suggestion
     * to a URL generated by `RouteOptions#toUrl`.
```

Could you also add a comment in `MapboxDirections#Builder.routeOptions` that unrecognized properties are not supported?

Reply:
added